packages/pipeline-editor/src/CustomFormControls/StringArrayControl.tsx

```diff
@@ -388,6 +399,16 @@ function StringArrayControl({ placeholder, format }: Props) {
               Browse
             </button>
           )}
+          {format === "properties" && (
+            <button
+              onClick={() => {
+                setEditingIndex(undefined);
+                handleNewValues();
+              }}
+            >
+              Detect Updates
```

Comment:
can we change this to `Refresh`? `Detect Updates` gives me the impression that it will keep automatically detecting updates

Reply:
done, will be in next commit